src/js/components/Wizard.js

```diff
@@ -1,22 +1,79 @@
 import React, {cloneElement} from 'react';
+import { browserHistory } from 'react-router';
 import werewolfService from '../services/werewolf';
 
+const DEFAULT_GAME_MODE = 'balanced';
 
 export default class Wizard extends React.Component {
   constructor(){
     super();
     this.state = {
       players: 0,
-      currentDeck: '',
+      currentDeck: 'basic', //todo: proper initialization with deck ' * '
+      currentCards: werewolfService.getCardsInDeck('basic'), // same as ^
+      mode: DEFAULT_GAME_MODE,
     };
   }
 
+  componentWillMount() {
+    this.setCurrentDeck(this.state.currentDeck);
+  }
+
   setPlayers(players) {
-    this.setState(Object.assign({}, this.state, players));
+    this.setState(Object.assign({}, this.state, {players}));
+  }
+
+  setCurrentDeck(currentDeck) {
+    const currentCards = werewolfService.getCardsInDeck(currentDeck)
+    .map(c => {
+      return {
+        key: c.key,
+        visible: werewolfService.isInDeck(c.key, currentDeck),
+        value: 1,
```

Comment:
Actually, that's required to be hardcoded (for now). That's the initial amount that a card will have. In order to remove this hardcode number we should have a centralized storage of cards amount (yes, this actually exists in the werewolf brain but the api doesn't return cards amounts).